pkg/deploy/che_configmap.go

```diff
@@ -67,7 +67,9 @@ type CheConfigMap struct {
 	WebSocketEndpointMinor               string `json:"CHE_WEBSOCKET_ENDPOINT__MINOR"`
 	CheWorkspacePluginBrokerInitImage    string `json:"CHE_WORKSPACE_PLUGIN__BROKER_INIT_IMAGE,omitempty"`
 	CheWorkspacePluginBrokerUnifiedImage string `json:"CHE_WORKSPACE_PLUGIN__BROKER_UNIFIED_IMAGE,omitempty"`
-	CheServerSecureExposesJwtProxyImage  string `json:"CHE_SERVER_SECURE__EXPOSER_JWTPROXY_IMAGE,omitempty"`
+	CheServerSecureExposerJwtProxyImage  string `json:"CHE_SERVER_SECURE__EXPOSER_JWTPROXY_IMAGE,omitempty"`
+	CheWorkspaceSidecarImagePullPolicy   string `json:"CHE_WORKSPACE_SIDECAR_IMAGE__PULL__POLICY,omitempty"`
+	CheDockerAlwaysPullImage             string `json:"CHE_DOCKER_ALWAYS__PULL__IMAGE,omitempty"`
```

Comment:
And probably override for [plugin_broker.pull_policy](https://github.com/eclipse/che/blob/36fe587e28fba1e37d9f6b38da673cf917a5132f/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L633) should be added

Reply:
maybe imagePullPolicy for jwtproxy should be overridden as well https://github.com/eclipse/che/blob/154d668505cc86763f16ad8868e0a1fcf4bccc28/infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/secure/jwtproxy/JwtProxyProvisioner.java#L287

Some of my messages may be out of the topic since I'm not familiar with airgap topic but I have seen familiar words and thought maybe by shared knowledge can be useful here =)

Reply:
After speaking with @nickboldt we had concluded that the airgap scenario doesn't need any overriding of the default pull policies. Even a `Always` pull policy would work in airgap mode, since we still point to a valid container registry.

So I'd better leave everything non-airgap-related out of the scope of this issue / PR.

Do you agree @sleshchenko ?

Reply:
BTW both properties:
```
CHE_WORKSPACE_SIDECAR_IMAGE__PULL__POLICY
CHE_DOCKER_ALWAYS__PULL__IMAGE
```
Have already removed by Tom.